### PR TITLE
Include more details to mr view

### LINKED ISSUE #322: Include more details in `glab mr view`
**Is your feature request related to a problem? Please describe.**
Currently, when looking at merge requests from glab, relevant details like if a merge request has been approved, the pipeline-status, or if it can be merged directly (rebase necessary?) are not easily visible.

**Describe the solution you'd like**
More details should be included in top status line. Here, it already mentions "merged" if a PR has been merged, so adding more details there would fit perfectly.
Additionally, it might be useful to include information about approval and pipeline-status in the `mr list` command, too.

### LINKED ISSUE #557: show pipeline status of the merge request
**Describe the feature or problem you'd like to solve**

When using `mr view` doesn't show me the status of the pipeline.

**Propose a Solution**

Show a green checkmark along with an identifier (ID) I can use to check the pipeline further via `glab pipeline view`

### PR DESCRIPTION
**Pull request by profclems**
**Description**
- feat(commands/mr/view): add pipeline status
- Include more details to MR view

**Related Issue**

Resolves #557, #322

**How Has This Been Tested?**

**Screenshots (if appropriate):**

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation
- [ ] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

File: commands/mr/view/mr_view.go [hunk 1]
```diff
@@ -155,6 +155,31 @@ func printTTYMRPreview(out io.Writer, mr *gitlab.MergeRequest, notes []*gitlab.N
 	if mr.State == "closed" {
 		fmt.Fprintf(out, "Closed By: %s %s\n", mr.ClosedBy.Username, mrTimeAgo)
 	}
+	if mr.Pipeline != nil {
+		fmt.Fprint(out, utils.Bold("Pipeline Status: "))
+		var status string
+		switch s := mr.Pipeline.Status; s {
+		case "failed":
+			status = utils.Red(s)
```
Comment (zemzale) [inline commands/mr/view/mr_view.go:163]: We had a change for ci status, that we printed pipeline status yellow, if it's allowed to fail 
like here https://github.com/profclems/glab/blob/8f5df7f1d0fbdfb9eb49d3c90af910c2db5668be/commands/ci/status/status.go#L84-L89

I would actually suggest that we bring out this pipeline status printing out, it's own helper function, so we can centralize it everywhere.

Comment (profclems) [reply commands/mr/view/mr_view.go:163]: Yeah you're right. I will do that

**Comment by profclems** [comment]
@zemzale do you have any suggestion as to how we can implement the approval status?

The [Approvals API](https://docs.gitlab.com/ee/api/merge_request_approvals.html) is only available for starter and bronze users so I'm hesitant to use it.

**Comment by zemzale** [comment]
I don't have any experience with the approval API or even that part of MRs, since I don't use it.

Saddly I am currently not really able to dig deeper into the topic either. So I cant help on this one

**Comment by profclems** [comment]
I think the approval status can't be part of v1.14.0 yet

**Pull request merged**